src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -404,6 +394,25 @@ internal Symbol BindNamespaceOrTypeOrAliasSymbol(ExpressionSyntax syntax, Diagno
             }
         }
 
+        private Symbol BindTupleExpression(TupleTypeSyntax syntax, DiagnosticBag diagnostics)
```

Comment:
Should probably be named BindTupleType or something like that that is more obvious it is dealing with tuple type. Otherwise it is too similar to the binding of actual tuple expression.